README.md

```diff
@@ -854,6 +855,139 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
+```javascript
+    var signIn = new OktaSignIn({
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}',
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+         registration: true
+      }
+    });
+```
+
+Following configurations are needed to enable registration on a self hosted sign in widget
+
+
+- **features.registration:** Used to enable registration feature on the widget.
+    ```javascript
+    // https://github.com/okta/okta-signin-widget#feature-flags
+    features.registration: 'true'
+
+    ```
+- **clientId:** Client Id pre-registered with Okta for the OIDC authentication flow
```

Comment:
We can probably remove both **clientId** and **features.registration** if we include this in the opening code snippet.

Reply:
Fixed